addon/lib/metrics.js

```diff
@@ -65,90 +59,70 @@ const variantMaker = {
 const Metrics = module.exports = {
 
   init: function() {
-    if (!store.telemetryPingPayload) {
-      store.lastTelemetryPingTimestamp = false;
-      store.telemetryPingPayload = {
-        version: 1,
-        tests: {}
-      };
-    }
-
-    Metrics.maybePingTelemetry();
-
+    store.browserLoadedTimestamp = Date.now();
```

Comment:
I & R's opinions are better than mine for this, but what you have here seems reasonable to me.